app/helpers/application_helper.rb

```diff
@@ -4,4 +4,16 @@ def categories
     Category.all
   end
 
+  def item_status_check(item)
```

Comment:
I can create a PORO if you want! This is the only method we need now but it could always take more